x-pack/plugins/rollup/public/crud_app/sections/components/job_details/tabs/tab_summary.js

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+
+import {
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiDescriptionListDescription,
+  EuiTitle,
+  EuiSpacer,
+  EuiIconTip,
+  EuiFlexItem,
+  EuiFlexGroup,
+} from '@elastic/eui';
+
+import { JobStatus } from '../../job_status';
+
+export class TabSummaryUi extends Component {
+  static propTypes = {
+    job: PropTypes.object.isRequired,
+    stats: PropTypes.object,
+  };
+
+  renderStats() {
+    const { stats } = this.props;
+
+    if (!stats) {
+      return null;
+    }
+
+    const {
+      documentsProcessed,
+      pagesProcessed,
+      rollupsIndexed,
+      triggerCount,
+      status,
+    } = stats;
+
+    return (
+      <Fragment>
+        <EuiSpacer size="l" />
+
+        <EuiTitle size="s">
+          <h3>
+            <FormattedMessage
+              id="xpack.rollupJobs.jobDetails.tabSummary.sectionStats.title"
+              defaultMessage="Stats"
+            />
+          </h3>
+        </EuiTitle>
+
+        <EuiSpacer size="s" />
+
+        <JobStatus status={status} />
+
+        <EuiSpacer size="s" />
+
+        <EuiDescriptionList textStyle="reverse">
+          <EuiFlexGroup>
+            <EuiFlexItem>
+              <EuiDescriptionListTitle>
+                <FormattedMessage
+                  id="xpack.rollupJobs.jobDetails.tabSummary.itemDocumentsProcessed.label"
```

Comment:
nit: To be consistent with other code, please follow to naming convention described in [Guideline](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md). Please use camelcase for the message type: `xpack.rollupJobs.jobDetails.tabSummary.itemDocumentsProcessedLabel`